src/pool/cli.c

```diff
@@ -827,7 +827,7 @@ dc_pool_l2g(daos_handle_t poh, d_iov_t *glob)
 	dc_pool_put(pool);
 out:
 	if (rc != 0)
-		D_ERROR("dc_pool_l2g failed, rc: "DF_RC"\n", DP_RC(rc));
```

Comment:
curious as to why make this change (for the three errors in this file)?

Reply:
One of these errors popped up at some point during development and I realised that the error string itself included the function name which is automatically added by D_ERROR() anyway, so I removed and searched for other instances in the same file.  Just trying to improve the quality of the debug output.